### PR TITLE
(/organizations/overview): update allow users to create organization path

### PR DESCRIPTION
**Pull request by NicolasLopes7**
> [!WARNING]
> Do not merge until https://github.com/clerk/dashboard/pull/3216 is merged

(/organizations/overview)

### CODE REVIEW

**Review by royanger — APPROVED** [review]

**Comment by kylemac** [comment]
linking https://github.com/clerk/dashboard/pull/3216 to this PR. lets not merge this one until we merge 3216

**Comment by alexisintech** [comment]
Adding `blocked` label so this doesn't accidentally get merged

**Pull request merged**
